src/components/Message.svelte

```diff
@@ -27,12 +29,13 @@
   class="message"
   class:inanimate
   class:thin
+  class:deleted
   style="display: {hidden ? 'none' : 'block'}"
 >
   <!-- For screenshot checkmark -->
   <slot />
 
-  {#each message.messageArray as msg}
+  {#each messageArray as msg}
```

Comment:
Was changing this from message.messageArray to messageArray necessary?

Reply:
Yup, the reactivity doesn't work if its kept as message.messageArray